CommunicationLayer/MessagingFramework/DriverControlDetails.cpp

```diff
@@ -12,7 +37,6 @@ namespace
    const int backEmfIndex = 13;
    const int dpsBoardTemperatureIndex = 17;
    const int dcBusAmpHoursIndex = 21;
-   const int odometerIndex = 25;
```

Comment:
Same comment as above, I don't think is a proper way of removing these fields from the protocol. 

If someone was looking for why the message size was a certain way they would be extremely confused since it was still receiving undocumented fields.